libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -155,22 +133,26 @@ $button-width: (
   // Outline is applied on button border instead,
   // to keep the rounded shape:
   outline: none;
-  border: 1px solid var(--kirby-button-border-color, transparent);
+  border-width: var(--border-width, 0px);
```

Comment:
Alternatively replace 0px with 1px and delete line 110.

Reply:
I think this might be due to some experiment not getting cleaned up properly 🙂

`--border-width` is necessary for calculating the `inset` value in the `layer` module (see your comment https://github.com/kirbydesign/designsystem/pull/2059#discussion_r821570110), so we can't delete line 110.